### PR TITLE
Update Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5a…

### PR DESCRIPTION
**Pull request by pvblick-vniversal-friend**
@luxumbra can you review this?

### CODE REVIEW

File: docs/playbooks/Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5ab.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# Path of the Writer Episode 1.0
+---
+title: 📚 Path of the Writer Episode 1.0
```
Comment (luxumbra) [inline docs/playbooks/Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5ab.md:2]: Thanks for adding this in @pvblick-vniversal-friend. 

We do need to fix the filename and add it to the `sidebar.js` file which is the classic wiki update gotcha! Not sure what happened with the file name but I expect the spaces will b0rk Docusaurus and we don't need the ` 0 617ae65ca624491ab516f40c8414b5ab` there. If you're not sure how to do that, I'm happy to help you do it or I can sort it in a jiffy.

**Review by luxumbra — CHANGES_REQUESTED** [review]

File: docs/playbooks/Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5ab.md (hunk 1 above)
Comment (pvblick-vniversal-friend) [reply docs/playbooks/Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5ab.md:2]: I'll try my hand at it and then I'll leave it to you to review it.

Comment (pvblick-vniversal-friend) [reply docs/playbooks/Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5ab.md:2]: @luxumbra done!

**Review by luxumbra — APPROVED** [review]
Cool, thanks for fixing those issues. LGTM. :)

We need to be mindful of XP earned from these kinds of updates and keep commits to a minimum otherwise the writers will end up gaining less XP than the ones reviewing the PRs.

**Pull request merged**